packages/shared/src/components/Feed.tsx

```diff
@@ -455,22 +454,6 @@ export default function Feed<T>({
     }
   };
 
-  const onAdAction = (
-    action: Exclude<AdActions, AdActions.Impression>,
-    ad: Ad,
-    row: number,
-    column: number,
-  ) => {
```

Comment:
Moved it down to FeedItemComponent so it can properly pickup extra context, it was actually only handler that was outside of FeedItemComponent.